documentation/operators/src/nodes/proxy-configuration.md

```diff
@@ -374,6 +378,261 @@ where you should read out
+
+1. Create a script by copying the block below and save it on your VPS as `install_run_nginx.sh`.
+
+```bash
+#!/bin/bash
+
+if [ "$#" -ne 2 ]; then
+    echo "usage: sudo ./install_run_nginx.sh <host_name> <port_to_run_wss>"
+    exit 1
+fi
+
+host=$1
+port_value=$2
+
+config_file_path="${HOME}/.nym/nym-nodes/*/config/config.toml"
+
+if [ ! -f $config_file_path ]; then
+    echo "configuration file not found at $config_file_path"
+    exit 1
+fi
+
+hostname=$(grep "hostname" $config_file_path | awk -F" = " '{print $2}' | tr -d "'")
+wss_port=$(grep "announce_wss_port" $config_file_path | awk -F" = " '{print $2}' | tr -d ' ')
+
+if [ -z "$hostname" ]; then
+    echo "hostname is empty, updating it to ${host}"
+    sed -i "s|hostname = ''|hostname = '${host}'|" $config_file_path
+else
+    echo "current hostname: $hostname"
+fi
+
+if [ "$wss_port" -eq 0 ]; then
+    echo "wss port is 0, updating it to ${port_value}"
+    sed -i "s/announce_wss_port *= *0/announce_wss_port = ${port_value}/" $config_file_path
+else
+    echo "current wss port: $wss_port"
+fi
+
+apt update
+apt install -y nginx
+
+sudo apt install certbot python3-certbot-nginx
+pip install --upgrade twine requests-toolbelt
+systemctl enable nginx.service
+
+if ! certbot certonly --nginx -d ${host} --non-interactive --agree-tos -m info@nymtech.net; then
+    echo "certbot failed to obtain certificates"
+    exit 1
+fi
+
+nginx_config_file="/etc/nginx/sites-available/${host}.conf"
+cat <<EOF > $nginx_config_file
+server {
+    listen ${port_value} ssl;
+    server_name ${host};
+
+    ssl_certificate /etc/letsencrypt/live/${host}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/${host}/privkey.pem;
+    include /etc/letsencrypt/options-ssl-nginx.conf;
+    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;
+
+    location / {
+        proxy_pass http://localhost:9000;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade \$http_upgrade;
+        proxy_set_header Connection "Upgrade";
+        proxy_set_header Host \$host;
+        proxy_cache_bypass \$http_upgrade;
+    }
+}
+EOF
+
+ln -s /etc/nginx/sites-available/${host}.conf /etc/nginx/sites-enabled/
+
+if ! nginx -t; then
+    echo "nginx configuration test failed"
+    exit 1
+fi
+
+systemctl reload nginx.service
+
+echo "script completed successfully!"
+echo "have a nice day!"
+exit 0
+```
+
+2. Make the script executable:
+```sh
+chmod u+x install_run_nginx.sh
+```
+
+3. Run the script as root (with `sudo` or from the root shell):
+```sh
+./install_run_nginx <HOSTNAME> <WSS_PORT>
+# hostname is your domain
+# wss default port is 9001
+```
+
+Your `nym-node` should be configured to run over WSS now. You can do a few quick checks to test that the setup worked out.
+
+- `wscat -c ws://<IP>:<WSS_PORT>`
+
+### Step by Step Guide
+
+
 <!--
```

Comment:
Is this a comment that was left in?

Reply:
Oh I wasn't finished, I should probably label it as draft meanwhile, I am sorry, but I will address your comments meanwhile, thank you.